eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionSender.java

```diff
@@ -6,6 +6,7 @@
 import com.microsoft.azure.eventhubs.impl.ExceptionUtil;
 
 import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ScheduledExecutorService;
```

Comment:
There was a Javadoc error about not being able to resolve the class.

Reply:
👍